lib/employees/resolvers.ts

```diff
@@ -119,3 +123,29 @@ export const updateEmployee: Resolver = async (_, args, { prisma }) => {
     ok: true,
   };
 };
+
+/**
+ * Delete an employee
+ * @returns status of operation (ok)
+ */
+export const deleteEmployee: Resolver = async (_, args, { prisma }) => {
+  const employee = await prisma.employee.findUnique({
+    where: {
+      id: parseInt(args.id),
+    },
+  });
```

Comment:
We can optimize this query by selecting only the fields we need (`active`). Please see https://www.prisma.io/docs/concepts/components/prisma-client/select-fields#select-specific-fields

Reply:
Also can we add some blank lines to this resolver for readability please?